Gutenberg/config.yml

```
@@ -1,14 +1,14 @@
+ # 'ref' should have either a commit or tag key.
+ # If both are set, tag will take precedence.
+ #
+ # We have automation that reads and updates this file.
+ # Leaving commented keys is discouraged, because as the automation would ignore them and the resulting updates would be noisy
```

Thanks @jhnstn. As a bonus, I noticed a mistake I made in the wording here:

```suggestion
 # Leaving commented keys is discouraged, because the automation would ignore them and the resulting updates would be noisy
```